spec/tags/truffle/arguments_descriptor_tags.txt

```diff
@@ -0,0 +1,2 @@
+fails:Arguments descriptors work for a mixture of arguments
+fails:Arguments descriptors work for a call like our Struct.new
```

Comment:
These two are legitimately failing specs for keyword argument behaviour in general - nothing to do with the descriptors.